workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowRunsTabContent.tsx

```diff
@@ -189,35 +191,37 @@ export const WorkflowRunsTabContent = () => {
         : [
             {
               title: 'Workflow name',
-              field: 'processName',
-              customSort: applyBackendSort,
+              field: 'name',
+              render: (data: WorkflowRunDetail) => (
+                <Link to={workflowPageLink({ workflowId: data.workflowId })}>
+                  {data.name}
+                </Link>
+              ),
             },
           ]),
       {
         title: 'Status',
         field: 'state',
-        render: (row: WorkflowRunDetail) => (
+        render: (data: WorkflowRunDetail) => (
           <WorkflowInstanceStatusIndicator
-            status={row.state as ProcessInstanceStatusDTO}
+            status={data.state as ProcessInstanceStatusDTO}
           />
         ),
-        customSort: applyBackendSort,
       },
       ...(workflowId
         ? []
         : [
             {
               title: 'Category',
               field: 'category',
-              render: (row: WorkflowRunDetail) =>
-                capitalize(row.category ?? VALUE_UNAVAILABLE),
-              sorting: false,
+              render: (data: WorkflowRunDetail) =>
+                capitalize(data.category ?? VALUE_UNAVAILABLE),
             },
           ]),
-      { title: 'Started', field: 'start', customSort: applyBackendSort },
-      { title: 'Duration', field: 'duration', sorting: false },
+      { title: 'Started', field: 'start', defaultSort: 'desc' },
+      { title: 'Duration', field: 'duration' },
```

Comment:
please return sorting: false
sorting is not possible for duration

Reply:
should I return customSort: applyBackendSort  as well?